docs-versioning.adoc

```diff
@@ -60,6 +56,62 @@ documentation releases to be updated independent of the KiCad release
 cycle, but still align with it, at the same time making it easy for
 the documentation maintainers call for a new release.
 
+== Branch Relations, Working Rules for Document
+
+KiCad application is developed on *lp:kicad* developement branch,
+and new functions for the future release are developed here.
+On the other hand, stable version of KiCad is maintained on stable
+branch (e.g. *lp:kicad/4.0*), and bugfix version of stable KiCad
+releases will come out from here.
+
+Having separate branches for devs and maintenance enables 2 different jobs
+independent.
+To follow this way of management, docs team also uses multi branches.
+
+The *master* branch is for docs developement and relates to *lp:kicad* KiCad
+application developement branch.
+We will update original English docs here.
+Docs stable branch (e.g. *4.0*) is for translation and relates to
+stable KiCad application branch (e.g. *lp:kicad/4.0*).
+We will manage localized stable docs here.
+
+```
+[[Application - launchpad]]
+<lp:kicad>     [+]--------------------[+]  -  -  -  - {developement}
+                |                      |
+                V                      V
+<stable>       [#]------$-----        [#]  -  -  -  - {bugfix, maintenance}
+               <lp:kicad/4.0>
+
+[[Documentation - GitHub]]
+<master>       [+]--------------------[+]  -  -  -  - {writers}
+                |                      |
+                V                      V
+<stable>       [#]------$-----        [#]  -  -  -  - {translators}
+               <4.0>    4.0.3         <5.0>
+```
+
+On *master* branch:
+
+- Writers update English .adoc files and related English menu screenshots.
+- Translation works are not accepted here, since those are invalidated soon
```

Comment:
I don't think we should blindly reject translations here if people want to translate, but we still want to inform them that the master reference document is volitile, so translations will probably be invalidated quickly.